tests/Rule/IsURLTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Tests\Morebec\Validator\Rule;
+
+
+use Morebec\Validator\Rule\isUrl;
+use PHPUnit\Framework\TestCase;
+
+class IsURLTest extends TestCase
+{
+    public function testValidate()
+    {
+        $rule = new IsURL();
+        $this->assertTrue($rule->validate('https://www.google.com'));
+        $this->assertFalse($rule->validate('a'));
```

Comment:
Although we rely on FILTER_VALIDATE_URL which should be reliable, it would be a good idea to add more tests for different types of URL, in case we ever decide to rely on some other validation logic:
Here are a few example that should pass:
```
[PASS] ftp://ftp.is.co.za.example.org/rfc/rfc1808.txt
[PASS] gopher://spinaltap.micro.umn.example.edu/00/Weather/California/Los%20Angeles
[PASS] http://www.math.uio.no.example.net/faq/compression-faq/part1.html
[PASS] mailto:mduerst@ifi.unizh.example.gov
[PASS] news:comp.infosystems.www.servers.unix
[PASS] telnet://melvyl.ucop.example.edu/
[PASS] http://www.ietf.org/rfc/rfc2396.txt
[PASS] ldap://[2001:db8::7]/c=GB?objectClass?one
[PASS] mailto:John.Doe@example.com
[PASS] news:comp.infosystems.www.servers.unix
```